Sources/WebDriver/Requests.swift

```diff
@@ -524,6 +524,55 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidtouchdoubleclick
+    public struct SessionTouchDoubleClick: Request {
+        public var session: String
+        public var element: String 
+
+        public var pathComponents: [String] { ["session", session, "touch", "doubleclick"] }
+        public var method: HTTPMethod {.post}
+        public var body: Body {.init(element: element)}
+
+        public struct Body: Codable {
+            public var element: String
+        }
+    }
+
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidtouchflick
+    public struct SessionTouchFlickExact: Request {
+        public var session: String 
+        public var element: String
+        public var xOffset: Int
```

Comment:
All these should be floats/doubles since the json API specifies `number` as the type. Particularly for the speed field.